modules/rds/rds-variables.tf

```diff
@@ -108,6 +108,11 @@ variable "iops" {
   default     = 0
 }
 
+variable "storage_throughput" {
+  type        = number
+  description = "The storage throughput value for the DB instance. Can only be set when `storage_type` is `gp3`. Cannot be specified if the `allocated_storage` value is below a per-engine threshold."
```

Comment:
please set the default value to match the default value provided in the module

Reply:
`default` added